src/classes/table/table.ts

```diff
@@ -146,9 +146,8 @@ export class Table implements ITable<any, IndexableType> {
     // and filter the rest.
     const { idxByName } = this.schema;
     const idb = this.db._deps.indexedDB;
-
     function equals (a, b) {
-      return idb.cmp(a,b) === 0; // Works with all indexable types including binary keys.
+      return cmp(a, b); // Works with all indexable types including binary keys.
```

Comment:
You mean `cmp(a, b) === 0;` ?

Reply:
dumb mistake on my part, fixed it. sorry about that